src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/OperationFormatter.cs

```diff
@@ -513,15 +513,15 @@ internal static void GetActions(OperationDescription description, XmlDictionary
             }
         }
 
-        internal static NetDispatcherFaultException CreateDeserializationFailedFault(string reason, Exception innerException)
+        public static NetDispatcherFaultException CreateDeserializationFailedFault(string reason, Exception innerException)
         {
```

Comment:
Can you move this method to be a static method on the NetDispatcherFaultException class